### PR TITLE
[Security] Remove Inner HTML Usage From Icon Creation

### PR DESCRIPTION
**Pull request by echen-adobe**
Describe your specific features or fixes

Changes the output of getIcon from a HTML string to a proper HTML tag, preventing devs from using innerHTML to set icons. Updates all usage cases of getIcon to accept this change.

Resolves: https://jira.corp.adobe.com/browse/MWPW-144969

Test URLs:
- Before: https://main--express--adobecom.hlx.page/express/
- After: https://create-tag-icon--express--adobecom.hlx.page/express/create/menu#how-to

App Banner:
    https://create-tag-icon--express--adobecom.hlx.page/docs/library/blocks/app-banner
Icon List
    https://create-tag-icon--express--adobecom.hlx.page//jp/express/create/video/vlog
Layouts
    https://create-tag-icon--express--adobecom.hlx.page/express/feature/image/resize
Premium Plan
    https://create-tag-icon--express--adobecom.hlx.page/docs/library/blocks/premium-plan
Quick Action Cards
    https://create-tag-icon--express--adobecom.hlx.page/docs/library/blocks/quick-action-cards
Ratings
     https://create-tag-icon--express--adobecom.hlx.page/docs/library/blocks/ratings
Toc
     https://create-tag-icon--express--adobecom.hlx.page/express/create/menu

Sticky-Footer
    Block removed,

### CODE REVIEW

**Review by JingleH — CHANGES_REQUESTED** [review]
Went through all the pages quickly and all icons behavior seem to be fine. The only thing I wasn't able to check was the app-banner. Can we also take care of the lint warning BTW? Otherwise all LGTM

**Comment by echen-adobe** [comment]
> Went through all the pages quickly and all icons behavior seem to be fine. The only thing I wasn't able to check was the app-banner. Can we also take care of the lint warning BTW? Otherwise all LGTM

Lint error should be taken care of now. I updated the app banner link with the correct one.

File: express/blocks/toc/toc.css [hunk 1]
```diff
@@ -40,10 +40,12 @@ main .toc-container .lottie {
 
 main .toc-container .default-content-wrapper .lottie {
   margin: -2px 0 -2px -8px;
+
```
Comment (qiyundai) [inline express/blocks/toc/toc.css:43]: nit: extra line break?

**Review by qiyundai — COMMENTED** [review]
just 2 non-blocking comments. LGTM otherwise

**Review by JingleH — APPROVED** [review]

**Review by qiyundai — APPROVED** [review]

**Pull request merged**